src/main/java/uk/gov/hmcts/darts/audio/service/impl/OutboundAudioDeleterProcessorImpl.java

```diff
@@ -66,7 +66,9 @@ public List<TransientObjectDirectoryEntity> markForDeletion() {
 
         List<TransientObjectDirectoryEntity> deletedValues = new ArrayList<>();
         for (TransientObjectDirectoryEntity entity : transientObjectDirectoryEntities) {
-            entity.getTransformedMedia().setExpiryTime(OffsetDateTime.now());
+            TransformedMediaEntity transformedMedia = entity.getTransformedMedia();
+            transformedMedia.setExpiryTime(OffsetDateTime.now());
+            transformedMedia.setLastModifiedBy(systemUser);
```

Comment:
Do we also need to set last modified ts?

Reply:
I think last modified ts is updated automatically.

Reply:
yep just confirmed that is the case